presto-docs/src/main/sphinx/release/release-0.202.rst

```diff
@@ -0,0 +1,78 @@
+=============
+Release 0.202
+=============
+
+General Changes
+---------------
+
+* Fix correctness issue for queries involving aggregations over the result of an outer join
+  (`issue #10592 <https://github.com/prestodb/presto/issues/10592>`_).
+* Fix :func:`map` to fail on duplicate keys, instead of producing a corrupted map silently.
+* Fix OOM in bucketed execution by scheduling new splits on the same worker as
+  the recently finished one.
+* Fix query failure when performing a ``GROUP BY`` on ``JSON`` and ``IPADDRESS`` types.
+* Fix :func:`map_from_entries` to throw ``INVALID_FUNCTION_ARGUMENT`` when input
+  array contains a ``null`` entry.
+* Fix correctness issue :func:`line_locate_point`, :func:`ST_IsValid`, and :func:`geometry_invalid_reason`
+  where they were returning values outside of the expected range.
+* Fix failure in :func:`geometry_to_bing_tiles` and :func:`ST_NumPoints` functions when
+  processing geometry collections.
+* Fix query failure in aggregation spilling
+  (`issue #10587 <https://github.com/prestodb/presto/issues/10587>`_).
+* Remove support for ``SHOW PARTITIONS`` statement.
+* Improve support for correlated subqueries containing equality predicates.
+* Improve performance of correlated ``EXISTS`` subqueries.
+* Limit the number of grouping sets in a ``GROUP BY`` clause.
+  The default limit is ``2048`` and can be set via the ``analyzer.max-grouping-sets``
+  configuration property, and ``max_grouping_sets`` session property.
+* Allow coercion between row types regardless of field names.
+  Previously, a row type is coercible to another only if the field name in the source type
+  matches the target type, or when target type has anonymous field name.
+* Increase default value for ``experimental.filter-and-project-min-output-page-size`` to 500kB.
+* Optimize array equals operator for ``bigint`` and ``double`` types.
+* Respect ``X-Forwarded-Proto`` header in client protocol responses.
+* Add support for column-level access control.
+  Connectors have not yet been updated to take advantage of this support.
+* Add support for correlated subqueries with correlated ``OR`` predicates.
```

Comment:
We had some other correlated subquery changes. These should group together. The "add support" and "improve support" seem to be the same way of stating the same thing.